libtransact/Cargo.toml

```diff
@@ -15,13 +15,15 @@ cbor-codec = "0.7"
 libc = ">=0.2.35"
 openssl = "0.10"
 uuid = { version = "0.7", features = ["v4"] }
+sawtooth-sdk = { git = "https://github.com/hyperledger/sawtooth-sdk-rust", optional = true }
```

Comment:
"0.3"?

Reply:
Currently, we do not publish `sawtooth-xo` to crates, and rust does not allow us to use the published sawtooth-sdk with the master version of sawtooth-xo.  It provides the following output:

```
error: multiple packages link to native library `zmq`, but a native library can be linked only once

package `zmq-sys v0.9.0`
    ... which is depended on by `zmq v0.9.0`
    ... which is depended on by `sawtooth-sdk v0.3.0`
    ... which is depended on by `transact v0.1.0 (/Users/pschwarz/Development/projects/transact/libtransact)`
links to native library `zmq`

package `zmq-sys v0.8.2`
    ... which is depended on by `sawtooth-zmq v0.8.2-dev5`
    ... which is depended on by `sawtooth-sdk v0.3.0 (https://github.com/hyperledger/sawtooth-sdk-rust#6df21f00)`
    ... which is depended on by `sawtooth-xo v0.8.0 (https://github.com/hyperledger/sawtooth-sdk-rust#6df21f00)`
    ... which is depended on by `transact v0.1.0 (/Users/pschwarz/Development/projects/transact/libtransact)`
```